.github/workflows/production_merge_validation.yml

```
@@ -0,0 +1,28 @@
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches:
+      - development
+
+jobs:
+  check-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check source branch name
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)